linux_os/guide/system/logging/package_rsyslog_installed/rule.yml

```diff
@@ -38,3 +38,5 @@ template:
     name: package_installed
     vars:
         pkgname: rsyslog
+    backends:
+        oval@rhel6: "off"
```

Comment:
The build system already has a preference for product specific implementations (`rhel6.xml` vs `shared.xml`).
So this is not strictly necessar.
But I like it as it makes explicit in the `rule.yml` that `rhel6` is not using the template implementation.